modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -286,6 +286,8 @@ const setupAttrHooks = (styles: Styles, settings: Partial<DOMUtilsSettings>, get
     style: {
       set: (elm, value) => {
         const sugarElm = SugarElement.fromDom(elm);
+        // TINY-9838: Prevent converting styles into shorthands for table element
+        const compressed = elm.nodeName !== 'TABLE';
```

Comment:
```suggestion
        const shouldCompress = elm.nodeName !== 'TABLE';
```
Maybe a variable name which describes its type and purpose?